tests/unit/generate_xml_test.py

```diff
@@ -46,3 +54,44 @@ def test_generate_complete_multipart_upload(self):
         ]
         actual_string = xml_marshal_complete_multipart_upload(etags)
         eq_(expected_string, actual_string)
+
+    def test_xml_marshal_select(self):
+        expected_string =   b'<SelectObjectContentRequest><Expression> select * from s3object</Expression>' \
+                            b'<ExpressionType>SQL</ExpressionType><InputSerialization>' \
+                            b'<CompressionType>NONE</CompressionType>' \
+                            b'<CSV><FileHeaderInfo>USE</FileHeaderInfo><RecordDelimiter>\n</RecordDelimiter><FieldDelimiter>,</FieldDelimiter>' \
+                            b'<QuoteCharacter>"</QuoteCharacter><QuoteEscapeCharacter>"</QuoteEscapeCharacter><Comments>#</Comments>' \
+                            b'<AllowQuotedRecordDelimiter>FALSE</AllowQuotedRecordDelimiter></CSV></InputSerialization>' \
+                            b'<OutputSerialization><CSV><QuoteFields>ASNEEDED</QuoteFields>' \
+                            b'<RecordDelimiter>\n</RecordDelimiter><FieldDelimiter>,</FieldDelimiter>' \
+                            b'<QuoteCharacter>"</QuoteCharacter><QuoteEscapeCharacter>"</QuoteEscapeCharacter></CSV></OutputSerialization>' \
+                            b'<RequestProgress><Enabled>TRUE</Enabled></RequestProgress></SelectObjectContentRequest>'
+
+
+        obj = SelectObjectOptions(
+            expression=" select * from s3object",
+            input_serialization = InputSerialization(compression_type="NONE",
+                                    csv=CSVInput(FileHeaderInfo="USE",
```

Comment:
Either vairables come together with `)` or `(` or enter such that all of them are indented appropriately.

Reply:
done.